### PR TITLE
Update USWDS to latest version (2.10.2)

### PR DESCRIPTION
**Pull request by aduth**
**Why**: So that we're taking advantage of the newest features and guidance, and to fix an incompatibility with older versions of normalize.css where summary element arrow marker is no longer shown in Chrome.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,32 +3,55 @@
...
+          paths:
+            - ~/.cache/yarn
+  build-release:
+    steps:
+      - run:
+          name: Create deploy.json
+          command: bundle exec rake login:deploy_json
+      - run:
+          name: Copy vendor dependencies
+          command: make copy_vendor
```
Comment (zachmargolis) [inline .circleci/config.yml:35]: we don't deploy from CI... is this just a matter of making sure the steps work & don't blow up?

File: Makefile [hunk 2]
```diff
@@ -9,10 +9,21 @@ PORT ?= 9292
...
 
+public/vendor:
+	mkdir -p public/vendor
+
+install_dependencies:
+	bundle check || bundle install
+	yarn install
+
+copy_vendor: public/vendor
+	cp -R node_modules/uswds/dist public/vendor/uswds
```
Comment (zachmargolis) [inline Makefile:23]: This is needed before deploying.

Can we update our handbook to describe that this step is needed? Or maybe add a `make prep_deploy` that calls this and `rake deploy:json` too? https://handbook.login.gov/articles/appdev-cloud-gov-deploy.html#deploying

**Review by zachmargolis — APPROVED** [review]
LGTM! Verified the `<details>` toggle is back, thank you!

File: .circleci/config.yml (hunk 1 above)
Comment (aduth) [reply .circleci/config.yml:35]: Hm, well that's a surprise to me! 😄 There's a `deploy` task here in the CircleCI configuration, so I thought we did. I'm not very familiar with this, so I'll give a look over the handbook link you shared below and see what changes might need to be made here.

Comment (aduth) [reply .circleci/config.yml:35]: This is addressed by #90

File: Makefile (hunk 2 above)
Comment (aduth) [reply Makefile:23]: Handbook documentation is updated in https://github.com/18F/identity-handbook/pull/100

**Pull request merged**